src/Facebook/Facebook.php

```diff
@@ -327,7 +350,7 @@ public function getRedirectLoginHelper()
    */
   public function getJavaScriptHelper()
   {
-    return new FacebookJavaScriptHelper($this->app);
+    return new FacebookJavaScriptHelper($this->app, $this->getOAuth2Client());
```

Comment:
Something like moving the `FacebookApp` dependency from the constructor to object methods :)